SF iOS/SF iOS/Feed/EventsFeedViewController.m

```diff
@@ -70,24 +75,76 @@ - (void)viewDidLoad {
     [self.tableView registerClass:self.feedItemCellClass forCellReuseIdentifier:NSStringFromClass(self.feedItemCellClass)];
     self.tableView.rowHeight = self.cellHeight;
     self.tableView.separatorStyle = UITableViewCellSeparatorStyleNone;
-    self.tableView.contentInset = UIEdgeInsetsMake(40, 0, 0, 0);
+    self.tableView.tableHeaderView.backgroundColor = UIColor.clearColor;
+    
+    UIEdgeInsets safeInsets = self.view.safeAreaInsets;
+    self.tableView.contentInset = UIEdgeInsetsMake(
```

Comment:
Stylistically I prefer having the first param on the same line in cases where the 2nd line ends to the right of the first line. I did a quick search and didn't see anything like this in the project. not a blocker.

Reply:
Done